test/test_c10d.py

```diff
@@ -2793,6 +2793,21 @@ def test_multi_limit_multi_dtype(self):
         result = dist._compute_bucket_assignment_by_size(tensors, [200, 400])
         self.assertEqual([[0], [1], [2, 4], [3, 5]], result)
 
+def per_process_setup(func):
+    '''
+        This function is to ensure we run some setup steps before the test
+        method is called on each process.
+    '''
```

Comment:
AFAIK the convention is to indent the comment block at the same level as the triple quote.